proto/cmp/services/activity/v1alpha/activity_types.proto

```diff
@@ -0,0 +1,200 @@
+syntax = "proto3";
+
+// ### Activity Services
+//
+// The Activity services are used for both tickets and excursions
+
+package cmp.services.activity.v1alpha;
+
+import "cmp/types/v1alpha/description.proto";
+import "cmp/types/v1alpha/location.proto";
+import "cmp/types/v1alpha/language.proto";
+import "cmp/types/v1alpha/file.proto";
+import "cmp/types/v1alpha/datetime_range.proto";
+import "cmp/types/v1alpha/duration.proto";
+import "cmp/types/v1alpha/contact_info.proto";
+import "cmp/types/v1alpha/product_code.proto";
+import "cmp/types/v1alpha/delivery.proto";
+import "cmp/types/v1alpha/redemption.proto";
+import "cmp/services/activity/v1alpha/search_result_types.proto";
+
+message ActivityExtendedInfo {
+    // Activity Product Details
+    ActivityDetails activity_product_detail_response = 1;
+
+    // Units
+    repeated ActivityUnit units = 2;
+
+    // Service Info
+    ActivityService service = 3;
+
+    // Zones
+    repeated ActivityZone zones = 4;
+
+    // Descriptions with different languages
+    repeated cmp.types.v1alpha.LocalizedDescriptionSet descriptions = 5;
+
+    // Coordinates
+    cmp.types.v1alpha.Coordinate position =6;
+
+    // Activity Features
+    repeated ActivityFeature features = 7;
+
+    // Tags
+    repeated ActivityTag tags = 8;
+
+    // Languages
+    repeated cmp.types.v1alpha.Language languages = 9;
+
+    // Contact Info; Address, Email, Phone, Urls
+    cmp.types.v1alpha.ContactInfo contact_info = 10;
+
+    // Images
+    repeated cmp.types.v1alpha.Image images = 11;
+
+    // Videos
+    repeated cmp.types.v1alpha.Video videos = 12;
+
+}
+
+message ActivityDetails {
+  // Context for Inventory system concepts that need to be included in an info response,
+  // like an OwnerCode, PTC_OfferParameters, Tax codes, Disclosure RefID, etc. or a
+  // serialized combination of these codes.
+  string context = 1;
+
+  // Errors encountered in the generation of the response. Standardized error list
+  // to be created.
+  repeated string errors = 2;
+
+  // Warnings encountered in the generation of the response. This section might include
+  // information about fields not processed due to incompatibilities with the
+  // inventory system
+  repeated string warnings = 3;
```

Comment:
@SamJaarsma Why do we have errors and warning here? Shouldn't those be in the header of the response message? We already have alerts there.

Reply:
Correct, my mistake, this is already in the ResponseHeader.alerts

Reply:
@havan @SamJaarsma we have the same thing in search,  check SearchResponseMetadata

Reply:
Those are wrong. We'll remove them with another PR.